nuxeo-core/nuxeo-core-event/src/test/java/org/nuxeo/ecm/core/work/WorkManagerTest.java

```diff
@@ -261,16 +265,19 @@ public void testWorkManagerScheduling() throws Exception {
     public void itCanFireWorkFailureEvent() throws Exception {
         int initCount = DummyPostCommitEventListener.handledCount();
         int initEvtCount = DummyPostCommitEventListener.eventCount();
-        
-        int duration = 5000; // 2s
+        int initSyncEvntCount = WorkFailureEventListener.getCount();
```

Comment:
you can/should use a tracker.assertDiff like for other tests it is more readable